draft-vvv-tls-alps.md

```diff
@@ -320,8 +331,11 @@ carried over from the previous connection.
 # IANA Considerations
 
 IANA will update the "TLS ExtensionType Values" registry to include
-`application_settings` with the value of TBD; the list of messages in which
-this extension may appear is `CH, EE, CEE`.
+`application_settings` with the value of 17613 (or 0x44cd); the list of messages
+in which this extension may appear is `CH, EE, CEE`.
+
+The ExtensionType value 17613 (or 0x44cd) is selected to replace the earlier
+experimental value 17513 (or 0x4469).
```

Comment:
The IANA considerations are instructions for IANA. This doesn't belong in IANA instructions.

If the aim here is to just document the experimental codepoints, you should leave this section as-is. Since you all haven't really started engaging with the standards process yet, it's far too early to declare 17613 as "the" codepoint here.

Reply:
revert the changes and keep the value as TODO